### PR TITLE
Bug 1977981: [External Mode] OCS Overview does not display any dashboard by default unless specific tab is clicked

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
**Before:**
Screenshot from 2021-07-01 01-42-11

**After:**
Working as expected now for the External Mode OCS.

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/dashboards/ocs-dashboards.tsx [hunk 1]
```diff
@@ -30,17 +32,32 @@ const OCSDashboardsPage: React.FC<DashboardsPageProps> = ({ match, kindsInFlight
...
+
+  const pluginPages = React.useMemo(() => {
+    const allTabs: (DashboardsTab | DynamicDashboardsTab)[] = [
+      ...tabExtensions,
+      ...dynamicTabExtensions,
+    ];
+    /** firstTabId === 'independent-dashboard', if it is an external mode OCS cluster.
+     *  firstTabId === 'persistent-storage', if it is not an external mode OCS cluster (internal mode).
+     *  firstTabId === 'object-service', if NooBaaSystem is there, but, StorageCluster is not.
+     */
```
Comment (GowthamShanmugam) [inline frontend/packages/ceph-storage-plugin/src/components/dashboards/ocs-dashboards.tsx:46]: @bipuladh Assuming this storage-> overview dashboard will be hidden when ODF is installed. This change is still required for OCS 4.8 and OCP 4.9 users.

Comment (GowthamShanmugam) [reply frontend/packages/ceph-storage-plugin/src/components/dashboards/ocs-dashboards.tsx:46]: Need to fix in the new upcoming storage system dashboard

**Comment by SanjalKatiyar** [comment]
closing this PR as this bug has already been taken care of in https://github.com/openshift/console/pull/9603

**Pull request closed without merging**